iOS Calculator/CalculatorEngine.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  CalculatorEngine.swift
+//  iOS Calculator
+//
+//  Created by Manas Paranjape on 12/15/18.
+//  Copyright © 2018 DesignX6. All rights reserved.
+//
+
+import Foundation
+
+var currentValue:Double = 0;
+var opJob:Double = 0;
+var operation:String = "";
+
+class CalculatorEngine {
+    struct Opwrapper {
+        var operand:Double
+        var operation:String
+    }
+    func clear() {
+        opJob = 0
+        operation = "";
+        currentValue = 0
+    }
+    struct OperandStack {
+        var items = [Double]()
+        mutating func push(_ item: Double) {
+            items.append(item)
+        }
+        mutating func pop() -> Double {
+            return items.removeLast()
+        }
+    }
+    var doubleStack = OperandStack()
+    func percenter (original: Double) -> Double {
+        return (original/100)
+    }
+    func posNeg(original: Double) -> Double {
+        let original2 = original * Double(-1)
+        return original2
+    }
+    func loadParameters(operand: Double, operationPassed: String = "") {
+        if currentValue == Double(0) {
+            operation = operationPassed
+            doubleStack.push(operand)
+            currentValue = operand
+            print(currentValue)
+        } else {
+            currentValue = doubleStack.pop()
+            print(currentValue)
+            if operation == "23" {
```

Comment:
Notice that if `operation` actually does equal "23", the code still checks whether `operation` equals all the other values too, which is unnecessary.  But instead of using `else` statements, _consider_ using a `switch`, like this:
```
switch operation {
case "23" :
    currentValue = add (...)
case "22" :
    currentValue = subtract (...)
case "21" :
    currentValue = multiply (...)
case "22" :
    currentValue = division (...)
default:
    // print error
}
```